libtransmission/rpc-server.cc

```diff
@@ -710,7 +708,7 @@ static bool bindUnixSocket(
     if (chmod(addr.sun_path, (mode_t)socket_mode) != 0)
     {
         tr_logAddWarn(
-            fmt::format(_("Could not set RPC socket mode to {mode:o}, defaulting to 755"), fmt::arg("mode", socket_mode)));
+            fmt::format(_("Could not set RPC socket mode to {mode:o}, defaulting to 0755"), fmt::arg("mode", socket_mode)));
```

Comment:
Since `{mode:o}` won't have a leading `0` (which it didn't in my local test, but maybe I missed something), maybe
```suggestion
            fmt::format(_("Could not set RPC socket mode to 0{mode:o}, defaulting to 0755"), fmt::arg("mode", socket_mode)));
```